File: Entidades/NotaPedidoEN.vb
Public Class NotaPedidoEN
    Private _codnot As Integer
    Public Property CodNot() As Integer
        Get
            Return _codnot
        End Get
        Set(ByVal value As Integer)
            _codnot = value
        End Set
    End Property

    Private _nronota As String
    Public Property NroNota() As String
        Get
            Return _nronota
        End Get
        Set(ByVal value As String)
            _nronota = value
        End Set
    End Property

    Private _codprov As Integer
    Public Property CodProv() As Integer
        Get
            Return _codprov
        End Get
        Set(ByVal value As Integer)
            _codprov = value
        End Set
    End Property


    Private _fecha As DateTime
    Public Property Fecha() As DateTime
        Get
            Return _fecha
        End Get
        Set(ByVal value As DateTime)
            _fecha = value
        End Set
    End Property

    Private _activo As Boolean
    Public Property Activo() As Boolean
        Get
            Return _activo
        End Get
        Set(ByVal value As Boolean)
            _activo = value
        End Set
    End Property

    Private _detalle As List(Of DetalleEN)
    Public Property Detalle() As List(Of DetalleEN)
        Get
            Return _detalle
        End Get
        Set(ByVal value As List(Of DetalleEN))
            _detalle = value
        End Set
    End Property
End Class

File: Entidades/NotaPedidoEN.cs
using System;
using System.Collections.Generic;

namespace Entidades
{
    public class NotaPedidoEN
    {
        private int _codnot;

        public int CodNot
        {
            get
            {
                return _codnot;
            }

            set
            {
                _codnot = value;
            }
        }

        private string _nronota;

        public string NroNota
        {
            get
            {
                return _nronota;
            }

            set
            {
                _nronota = value;
            }
        }

        private int _codprov;

        public int CodProv
        {
            get
            {
                return _codprov;
            }

            set
            {
                _codprov = value;
            }
        }

        private DateTime _fecha;

        public DateTime Fecha
        {
            get
            {
                return _fecha;
            }

            set
            {
                _fecha = value;
            }
        }

        private bool _activo;

        public bool Activo
        {
            get
            {
                return _activo;
            }

            set
            {
                _activo = value;
            }
        }

        private List<DetalleEN> _detalle;

        public List<DetalleEN> Detalle
        {
            get
            {
                return _detalle;
            }

            set
            {
                _detalle = value;
            }
        }
    }
}